src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,35 @@
+.watch {
+  display: inline-block;
+  width: 10px;
```

Comment:
use variables for the main values so that you'll be able to reuse them and give them descriptive names